### ISSUE TITLE
Proposal peer review

### ISSUE THREAD

**Issue by SaikumarBollam** [INFO-526-FA24/project-2-insight-spark#2]
The following is the peer review of the project proposal by project-2-insight-spark. The team members that participated in this review are

1. **Full Names and GitHub Names of Reviewing Team:**
   - **Reviewer 1:** Saikumar Bollam
   - **Reviewer 2:** Vamsi Krishna C

2. **Describe the Goal of the Project:**
   - The goal of the project is to analyze unemployment trends across U.S. states from 1976 to 2022. The team aims to uncover patterns in unemployment rates by state and over time, especially in response to major economic events such as the 2008 financial crisis and the COVID-19 pandemic. This analysis will provide insights into which states show resilience or vulnerability in the face of economic fluctuations.

3. **Describe the Data Used or Collected:**
   - The data used is the “Unemployment in America, Per US State” dataset from Kaggle. It contains state-wise unemployment statistics from 1976 to 2022, including total employment, labor force size, and unemployment rate by state. Key columns include FIPS codes, state names, year, month, and other metrics such as employment and unemployment counts and rates.

4. **Describe How the Research Question Will Be Answered, e.g., What Approaches/Methods Will Be Used:**
   - The team plans to perform data cleaning and preprocessing to standardize the dataset and handle missing values. Exploratory Data Analysis (EDA) will include summary statistics for each state and identification of patterns in unemployment rates over time. Visualizations, such as line plots of unemployment trends and animated choropleth maps, will illustrate changes in unemployment across states and during significant economic events.

5. **Is There Anything That Is Unclear from the Proposal?**
   - The proposal is generally clear, but some aspects of the analytical approach could benefit from more detail. Specifically, it would help if the proposal clarified which statistical methods (e.g., regression analysis, clustering) will be used to analyze trends and correlations. Additionally, it’s unclear how the team will handle missing values or whether any data imputation methods will be applied.

6. **Provide Constructive Feedback on How the Team Might Be Able to Improve Their Project:**
   - It may be beneficial to include a brief explanation of how major economic events will be identified or marked in the analysis (e.g., using date ranges for events like the 2008 financial crisis).

7. **What Aspect of This Project Are You Most Interested in and Would Like to See Highlighted in the Presentation?**
   - Highlighting periods of economic downturns and how different regions responded would be very insightful, especially if specific states that demonstrate resilience are discussed.

8. **Provide Constructive Feedback on Any Issues with File and/or Code Organization:**
   - The code for renaming columns and loading data is organized, but be sure to include comments for each step to ensure readability. 
   - It would also be useful to add more documentation or a README file in the GitHub repository, detailing the project structure, dependencies, and how to run the code.

9. **(Optional) Any Further Comments or Feedback?**
   - Overall, this is a well-thought-out project with a strong focus on uncovering regional trends in unemployment. It could be beneficial to add a brief section on potential policy implications of the findings, as this might add depth to the analysis and showcase the broader impact of the work.

**Comment by nithingm** [comment]
We've made the following changes:
1. More details to how the EDA would be conducted
2. More details on the statistical methods to analyze trends and correlations
3. We will be focusing on a few global events, namely 2008 financial crisis and Covid 2019. Once the analysis is done, we'll be possibly looking at more events.
4. README file has been added.
5. Comments will be added to more code that arises.
6. We don't need to deal with null values or missing data because the dataset is already cleaned.
